source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -382,6 +384,14 @@ information:
    Any other document validation instances will cause the client to throw
    an error.
 
+.. note::
+
+   You can prevent clients that are not configured with the appropriate
+   client-side JSON Schema from writing unencrypted data to a field by
+   using server-side document validation. For server-side document validation
+   implementation details, see :manual:`Enforce Field Level Encryption Schema
+   </core/security-client-side-encryption/#enforce-field-level-encryption-schema>`.
```

Comment:
- Could give this note a title: **Server-side JSON Schema**
- unencrypted data OR data that is improperly encrypted
- Maybe add a sentence that explicitly says that this is only supplemental to CSFLE

Reply:
> Could give this note a title: Server-side JSON Schema
Sure. I avoided that term because: 
1) It hasn't been declared as the de facto term in the server manual.
2) It could cause confusion since the section title doesn't specify which type (server-side or client-side) of schema we're creating in this step

> unencrypted data OR data that is improperly encrypted
Could you explain what you mean by "improperly encrypted"? Do you mean that the client-side JSON Schema references the wrong key? I wonder if there is a way to make this more general to encompass all the ways it could go wrong as it would otherwise require listing out that info.

> Maybe add a sentence that explicitly says that this is only supplemental to CSFLE
👍

Reply:
I think this does not include "improperly encrypted" for the following reason:

> Since the MongoDB server cannot decrypt nor introspect the contents of the encrypted field, it cannot validate that clients used the specified encryption options to encrypt a given field. This allows two clients to insert encrypted data using different keyIDs or encryption algorithms for a specific field. While some workloads may require independent field level encryption implementations, inconsistent implementation of encryption options for a field across clients may result in incorrect or unexpected behavior of queries against the encrypted field.